Tests/scripts/hook_validations/id.py

```diff
@@ -163,7 +163,8 @@ def is_id_duplicated(self, obj_id, obj_data, obj_type):
                             is_duplicated = True
                             break
 
-                    elif LooseVersion(obj_fromversion) <= LooseVersion(instance_to_version):
+                    elif not (LooseVersion(obj_fromversion) > LooseVersion(instance_to_version)
```

Comment:
Lets change it to <= and also change it to and condition instead

Reply:
@ronykoz an `and` condition means that either both of them are wrong or both of them are ok. The first case is impossible, and the second case means `is_duplicated` should be false. So I don't see how we can make it an `and` condition.

Reply:
You’ve done not(a or b) which means not a and not b, it would be more readable

Reply:
ok, and why do we need the `=`?

Reply:
Because you want to make sure the versions don’t collide, we must have a gap between them otherwise they both turn up as valid, and you won’t be able to tell which of them was loaded

Reply:
but it's a `not`, so it's the other way around